app/views/profiles/show.html.erb

```diff
@@ -29,7 +29,9 @@
   </div>
 
   <div class="flex gap-6 justify-center">
-    <button class="btn btn-sm">View Resume</button>
+    <% if @profile.user.current_resume %>
```

Comment:
I considered showing the button no matter what, but decided on this implementation since I think it's less confusing to only show a method to view a resume if it's possible, rather than show a disabled button and provide instructions on how to enable it.